src/pip/_internal/commands/search.py

```diff
@@ -111,9 +112,9 @@ def transform_hits(hits: List[Dict[str, str]]) -> List["TransformedHit"]:
     return list(packages.values())
 
 
-def print_dist_installation_info(name: str, latest: str) -> None:
-    env = get_default_environment()
-    dist = env.get_distribution(name)
+def print_dist_installation_info_if_exists(
```

Comment:
There's no real need to add `_if_exists` to the name - `print_dist_installation_info(latest, dist)` seems perfectly understandable to me. Apart from this minor point, this looks good, thanks.